00_make_block_group_shp.R

```diff
@@ -35,7 +35,11 @@ saveRDS(blk_grps_sf_1990, "block_groups_1990_5072.rds")
 tracts_sf_1980 <- st_read('/Users/RASV5G/Downloads/nhgis0018_shape/nhgis0018_shapefile_tl2000_us_tract_1980/US_tract_1980.shp')
 
 tracts_sf_1980 <- st_transform(tracts_sf_1980, crs=5072) %>%
-  dplyr::select(fips_tract_id_1980 = GISJOIN2,
+  dplyr::mutate(state_fips = stringr::str_sub(NHGISST, 1, 2),
+                county_fips = stringr::str_sub(NHGISCTY, 1, 3),
+                tract_fips = stringr::str_sub(GISJOIN, 9),
```

Comment:
I think this is implying `end = -1` here, but can we explicitly state `end=14` (or whatever it would be for the tract fips)?

Reply:
I did it this way because some of the tract fips in 70 and 80 are 4 digits and some are 6, so they have different lengths. Is this what you wanted me to pad?

Reply:
Yes, I think we should probably make all census tract IDs six digits.